### PR TITLE
feat: add layer zero tx link

### PR DESCRIPTION
**Pull request by alter-eggo**
# Description
See title

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Review by garethfuller — COMMENTED** [review]
Where do you clear the sync tx hashes once it's complete? i.e. what happens if they sync, then extend the lock again, and then try syncing again. Will the tx hashes from the first sync be mixed up with the second?

**Comment by alter-eggo** [comment]
sync tx hash is set once user submit tx on any network, they cannot mix up

File: src/components/contextual/pages/vebal/cross-chain-boost/SyncNetworkAction.vue [hunk 1]
```diff
@@ -75,7 +77,9 @@ async function handleAction(network: Network) {
       'tempSyncingNetworks',
       JSON.stringify(tempSyncingNetworks.value)
     );
-
+    setTimeout(() => {
+      setSyncTxHashes(network, tx.hash);
+    }, 5000);
```
Comment (garethfuller) [inline src/components/contextual/pages/vebal/cross-chain-boost/SyncNetworkAction.vue:82]: I think this should be tried more than once. Can call on an interval after the tx success? Maybe allow it to be called 3-5 times every 10s?

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: src/components/contextual/pages/vebal/cross-chain-boost/SyncNetworkAction.vue (hunk 1 above)
Comment (alter-eggo) [reply src/components/contextual/pages/vebal/cross-chain-boost/SyncNetworkAction.vue]: done

**Review by garethfuller — APPROVED** [review]

**Pull request merged**